io-manager.js

```diff
@@ -64,8 +64,8 @@ ioManager.keys = {
   space: false,
   ctrl: false,
 };
-let lastShiftDownTime = 0;
-ioManager.getLastShiftDownTime = () => lastShiftDownTime;
+let lastWASDDownTime = [0,0,0,0];

```

Comment:
I would label this with keys, not mysterious indices.

Reply:
Changing that now